Docker/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM node:16-alpine
+FROM node:17-alpine3.14
 WORKDIR /app
 COPY . .
 RUN apk add --no-cache --virtual .build-deps make g++ \
-    && apk add --no-cache python3 \
+    && apk add --no-cache git bash python3 \
```

Comment:
we don't want git in the container because the code should not be changing. these containers are aimed at production deploys.

Bash is also not necessary.

Reply:
Indeed. I had a complain about not finding these 2 when building locally. Will doublecheck the commands here